app/views/payments/new.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="new-payment">
+  <h1>New Payment</h1>
+  <%= form_with model: @payment, url: category_payments_path do |f| %>
+    <%= f.text_field :name, placeholder: 'Enter Full Name' %>
+    <br>
+    <br>
+    <%= f.text_field :amount, placeholder: 'Enter Amount' %>
+    <br>
+    <br>
+    <%= f.collection_check_boxes :category_ids, Category.where(author_id: 1), :id, :name do |cat| %>
+      <div class="checkbox-div" >
+        <%= cat.check_box %>
+        <%= cat.label %>
+      </div>
+    <% end %>
+    <br>
+    <%= f.submit('Add Payment') %>
+  <%end%>
+</div>
```

Comment:
- [ ] Kindly present the user with error messages when they submit invalid details, instead of doing nothing.

- [ ] This Payment form page should have a Back (<) for navigating back to the category show page.

Reply:
Fixed